packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -365,6 +366,7 @@ export async function renderConfirmationPrompt({
 export interface RenderAutocompleteOptions<T>
   extends PartialBy<Omit<AutocompletePromptProps<T>, 'onSubmit'>, 'search'> {
   renderOptions?: RenderOptions
+  groupOrder?: string[]
```

Comment:
let's move the `groupOrder` to the `AutocompletePromptProps`, seems like that's where all the props for this element are